system/mauveLoadTest/playlist.xml

```diff
@@ -11,6 +11,18 @@
 				<impl>ibm</impl>
 				<version>8</version>
 			</disable>
+			<disable>
+				<comment>https://github.com/eclipse-openj9/openj9/issues/22912</comment>
+				<version>17</version>
+			</disable>
+			<disable>
+				<comment>https://github.com/eclipse-openj9/openj9/issues/22912</comment>
+				<version>21</version>
```

Comment:
I think the two disables above can be combined into one `<version>[17,21]</version>`